design-docs/build.js

```
@@ -0,0 +1,83 @@
+import { readFileSync, writeFileSync, existsSync } from 'node:fs';
+import { fileURLToPath } from 'node:url';
+import { dirname, resolve, basename } from 'node:path';
+import { parseBarrel, extractStatus, mergeManifest } from './manifest.js';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)